tools/common.ps1

```diff
@@ -143,6 +143,28 @@ function Get-EbpfPackageUrl {
     return "https://github.com/microsoft/ebpf-for-windows/releases/download/Release-v" + $EbpfVersion + "/" + $EbpfPackageName
 }
 
+function Get-FnVersion {
+    return "0.4.1"
```

Comment:
This seems painful to keep in-sync with the dependency. Is there no better way to have this somewhere more easily found?

Reply:
I 100% agree. This is the pattern used for the eBPF dependency though (and the pattern used by the wtc project's consumption of fnmp), so its no worse off than how existing dependencies are handled.

Reply:
Alright. We can keep it as is for now. But I'm not a big fan. I'd prefer some top-level json file or something that lists the dependencies and version numbers. We can follow up later though.